client/src/components/transaction-list/transaction-list.vue

```diff
@@ -16,6 +18,8 @@ const terminalEl = ref<Element>();
 const transactionStore = useTransactionsStore();
 const { userTransactions, botTransactions } = storeToRefs(transactionStore);
 
+const channel = useChannelStore().channel;
+
```

Comment:
```suggestion
const channel = useChannelStore().channel;
const didReconnect = !!localStorage.getItem('gameState');
```

No need for it to be ref as we will only check it on first render